cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/contactpoint/HttpClusterBootstrapRoutes.scala

```diff
@@ -39,17 +36,10 @@ final class HttpClusterBootstrapRoutes(settings: ClusterBootstrapSettings) exten
     }
   }
 
-  // TODO ip whitelist feature?
-  val routes = {
-    // TODO basePath, same as akka-management
-    // val basePath = if (pathPrefixName.isEmpty) rawPathPrefix(pathPrefixName) else pathPrefix(pathPrefixName)
-
+  val routes =
     toStrictEntity(1.second) { // always drain everything
```

Comment:
Isn't this dangerous, given that we will get concatenated with other routes, what if this route goes before others?

Reply:
Will increase a bit

Reply:
Ah right, I moved it into the matched route